backend/app/services/__init__.py

```
@@ -0,0 +1,7 @@
+"""Services package initialization."""
+
+from app.services.base import BaseService
+from app.services.item import ItemService
+from app.services.user import UserService
+
+__all__ = ["BaseService", "UserService", "ItemService"]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)